src/main/java/wolox/training/models/Book.java

```diff
@@ -0,0 +1,123 @@
+package wolox.training.models;
+
+import javax.persistence.Column;
+import javax.persistence.Entity;
+import javax.persistence.GeneratedValue;
+import javax.persistence.GenerationType;
+import javax.persistence.Id;
+
+@Entity
+public class Book {
+  @Id
+  @GeneratedValue(strategy = GenerationType.AUTO)
+  private long id;
+
+  private String genre;
+
+  @Column(nullable = false)
+  private String author;
+
+  @Column(nullable = false)
+  private String image;
+
+  @Column(nullable = false)
+  private String title ;
+
+  @Column(nullable = false)
+  private String subtitle;
+
+  @Column(nullable = false)
+  private String publisher;
+
+  @Column(nullable = false)
+  private String year;
+
+  @Column(nullable = false)
+  private Integer pages;
+
+  @Column(nullable = false)
+  private String isbn;
+
+  public Book() {
+  }
+
+  public long getId() {
+    return id;
+  }
+
+  public void setId(long id) {
+    this.id = id;
+  }
```

Comment:
This setter should not be included, because the id is generated automatically and we don't want it to be changed from other classes :)

Reply:
Got it! I've pushed the changes :)